src/docs/product/alerts/index.mdx

```diff
@@ -44,6 +44,16 @@ You can find a full list of available metric alerts in [Metric Alerts](/product/
 
 When you create a new project in [sentry.io](https://sentry.io), you can select a default issue alert. However, you can also [create your own alerts](/product/alerts/create-alerts/) to suit your team’s needs, using these [best practices](/product/alerts/best-practices/) as a guide.
 
+## Muting Alerts
+
+<Note>
+
+By default all users can mute alerts for everyone. Sentry users with manager or owner permissions can change the minimum role requirement in **Organization Settings > General Settings > Let Members Create and Edit Alerts**.
```

Comment:
I put "organization settings" here but I noticed we don't explicitly say this elsewhere in the docs, we just call it "settings" which seems wrong? We have org and user level settings, and nothing just called settings ...

Reply:
Nvm just noticed the " ⚙️ Settings" thing on the sidebar, idk if it's new or I've literally never noticed it before cause I always click the dropdown and then org settings.

Reply:
I think "settings" might just come from the sidebar which says settings but takes you to org settings.

Reply:
Yeah I am losing my mind here feeling like a mandela effect is happening except I'm the only person who thinks we never had ⚙️ Settings before.